xonsh/parsers/fstring_adaptor.py

```diff
@@ -0,0 +1,182 @@
+import re
```

Comment:
Please add a module level docstring as well as a hook in the `docs/` dir as well.

Reply:
I don't see where this fits within `docs/`.

Reply:
It's OK! I'll add after merge

Reply:
Oh weird, I guess we don't have parser docs linked in sphinx yet